docs/build/guides/conventions/workspace.mdx

```diff
@@ -0,0 +1,357 @@
+---
+title: Workspaces
+hide_table_of_contents: true
+description: Organize contracts using Cargo workspaces
+---
+
+### Initializing workspace project
+
+Using Cargo's workspace [feature](https://doc.rust-lang.org/book/ch14-03-cargo-workspaces.html) makes it very convenient to organize your smart contracts in subdirectories of your project's root.
+
+It's very simple to get started using the cli:
+
+```
+stellar contract init . --name add_contract
+```
+
+Running this command will create a root project directory (`soroban-project`) and then initialize a project workspace with a single contract named `add_contract`.
+
+Adding one more contract template to the project can be done using the same command:
+
+```
+stellar contract init . --name main_contract
+```
+
+The project tree will look like this:
+
+```
+$ tree
+.
+├── Cargo.toml
+├── contracts
+│   ├── add-contract
+│   │   ├── Cargo.toml
+│   │   ├── Makefile
+│   │   └── src
+│   │       ├── lib.rs
+│   │       └── test.rs
+│   └── main-contract
+│       ├── Cargo.toml
+│       ├── Makefile
+│       └── src
+│           ├── lib.rs
+│           └── test.rs
+└── README.md
+```
+
+Running `stellar contract init` command created a sample contracts subdirectories (located in `./contracts), each contains:
+
+1. Cargo.toml file with all (minimal) necessary dependencies
+2. `Makefile` to easily build contract using `make` command
```

Comment:
I think we should probably not call attention to this. Here we signal about using the make command for it's ease. Then none of the rest of the guide makes use of it. Given the make command will only be appreciated by a subset of users we probably do not need to dedicate text to it which may be distracting to new devs.
```suggestion
```